src/core/frame.c

```diff
@@ -407,10 +407,13 @@ meta_frame_sync_to_window (MetaFrame *frame,
 cairo_region_t *
 meta_frame_get_frame_bounds (MetaFrame *frame)
 {
-  return meta_ui_get_frame_bounds (frame->window->screen->ui,
+  if(frame->xwindow && frame->window !=NULL && frame->window->screen  !=NULL && frame->window->screen->ui !=NULL)  {
+	return meta_ui_get_frame_bounds (frame->window->screen->ui,
                                    frame->xwindow,
```

Comment:
Can all following lines starts under `frame->` after the opening bracket please?

Reply:
`frame->xwindow,` and othe lines of this function should start under
`frame->window->screen->ui,`
Is this possible?

Reply:
OMG, complete wrong. I meant this
```
return meta_ui_get_frame_bounds (frame->window->screen->ui,
                                   frame->xwindow,
                                   frame->rect.width,
                                   frame->rect.height);
  }
```